docs/how-to-guides/working-with-images.md

```diff
@@ -1,7 +1,11 @@
+---
+title: Working with Images
+---
+
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Images
+# Working with Images
```

Comment:
this line is redundant as the title comes form the header meta information and hence the page prints the title twice.